ruma-federation-api/src/discovery/discover_homeserver.rs

```diff
@@ -17,8 +17,8 @@ ruma_api! {
     request: {}
 
     response: {
-        /// The server name to delegate server-server communciations to, with optional port.
-        #[serde(rename = "m.homeserver")]
+        /// The server name to delegate server-server communications to, with optional port.
+        #[serde(rename = "m.server")]
         pub homeserver: ServerNameBox,
```

Comment:
what do you think about renaming also this property to `pub server`?

Reply:
Makes sense to me.

Reply:
Renamed here: https://github.com/ruma/ruma/pull/542/commits/b7122138c50c17393b93b6a3241b8ee60b135116